IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/Browser.cs

```diff
@@ -145,22 +146,33 @@ public void TakeScreenshot()
         /// <param name="file">The fully qualified or relative name of the file to save the screenshot.</param>
         public void TakeScreenshot(FileInfo file)
         {
+            if(file == null)
+            {
+                Debug.WriteLine($"Skipping TakeScreenshot. Argument {nameof(file)} was null.");
+                return;
+            }
+
             if (string.IsNullOrWhiteSpace(file.FullName))
             {
-                Console.WriteLine($"Skipping TakeScreenshot. FileInfo handed into method had no valid full name.");
+                Debug.WriteLine("Skipping TakeScreenshot. FileInfo handed into method had no valid full name.");
                 return;
             }
 
             if (!file.Directory.Exists)
             {
-                Console.WriteLine($"Skipping TakeScreenshot. Path does not exist: {file.FullName}");
+                Debug.WriteLine($"Skipping TakeScreenshot. Path does not exist: {file.FullName}");
                 return;
             }
 
+            if(file.Exists)
```

Comment:
`file.Delete()` does nothing if the file doesn't exist, so you don't need this check here.

Reply:
Removed the check.